### ISSUE TITLE
Plotting list object from sliced mesh

### ISSUE THREAD

**Issue by Huzaifi96** [pyvista/pyvista-support#113]
Hi,

I want to plot the sliced meshes from an object(polydata). I'm using the add_mesh_slice method from the PyVista widget to get sliced meshes. However, the sliced meshes return as a list object. I can't plot the sliced meshes using pyvista.Plotter since the sliced meshes is a list types object. Is there any method that I can use to convert the list object to the PyVista polydata object so that I could plot the 2D plane of sliced meshes 

Here is the code:

`import pyvista as pv`
`import numpy as np`

`vol = pv.PolyData('C:/Users/huzai/Desktop/hms ff.stl')`
`p = pv.Plotter(notebook=False)`
`p.add_mesh_slice(vol) # Slice a mesh using a plane widget.`
`p.show()`
`w = p.plane_sliced_meshes # return sliced meshes (list types)`
`print(w)`

from `p.show()`: 

![sliced meshes](https://user-images.githubusercontent.com/58688246/73356568-a1e79f00-42d5-11ea-963a-02d790222e9c.jpg)

from `print(w)` :

`[PolyData (0x19c4936e1c8)
  N Cells:	2860
  N Points:	2860
  X Bounds:	1.404e+00, 1.404e+00
  Y Bounds:	5.906e+01, 1.104e+02
  Z Bounds:	1.076e+02, 2.812e+02
  N Arrays:	0
]`

**Comment by banesullivan** [comment]
Yep, the `plane_sliced_meshes` is a list of meshes as you can have more than one slicing widget in the scene and the order in the list corresponds to the order in which the sliders were added to the scene.

To do something with those meshes, simply grab the mesh from the list:

```py
# for first plane widget's mesh:
mesh = w[0]
mesh.plot()
```

Or maybe you have a whole bunch of widgets each producing meshes, then you could make a `MultiBlock` dataset of all the clipped meshes:

```py
slices = pv.MultiBlock(w)
slices.plot()
```


------

this could be a feature request to just use a `MultiBlock` dataset instead of a list in those widgets methods. Would be a trivial patch.

**Comment by Huzaifi96** [comment]
Yeah, I got it. Thanks for the help @banesullivan
